.travis.yml

```diff
@@ -1,31 +1,37 @@
 language: php
 
+dist: trusty
+
 env:
   global:
-    - COMPOSER_ROOT_VERSION=2.0.x-dev
+    - COMPOSER_ROOT_VERSION=4.0.x-dev
 
 matrix:
   include:
     - php: 5.6
       env: DB=MYSQL PHPCS_TEST=1 PHPUNIT_TEST=1
     - php: 7.0
+      env: DB=PGSQL PHPUNIT_TEST=1
+    - php: 7.0
       env: DB=MYSQL PHPUNIT_TEST=1
     - php: 7.1
       env: DB=MYSQL PHPUNIT_COVERAGE_TEST=1
 
 before_script:
+  # Init PHP
   - phpenv rehash
   - phpenv config-rm xdebug.ini
 
-  - composer install --prefer-dist
-  - composer require --prefer-dist --no-update silverstripe/installer 4.0.x-dev
-  - if [[ $DB == PGSQL ]]; then composer require --prefer-dist --no-update silverstripe/postgresql 2.0.x-dev; fi
-  - composer update
+  # Install composer dependencies
+  - composer validate
+  - composer require silverstripe/recipe-core 1.0.x-dev  --no-update
```

Comment:
I assume we need the CMS recipe for this module? Also relevant for the bootstrap path in phpunit.xml.dist